docs/zh/latest/plugins/basic-auth.md

```diff
@@ -21,128 +21,154 @@ title: basic-auth
 #
 -->
 
-## 目录
+## 简介
 
-- [**名字**](#名字)
-- [**属性**](#属性)
-- [**如何启用**](#如何启用)
-- [**测试插件**](#测试插件)
-- [**禁用插件**](#禁用插件)
+启用该插件后，客户端访问路由、服务时需提供正确的用户名与密码，插件将从 HTTP 请求头 Authorization 中获取凭证信息。
 
-## 名字
+:::caution 注意
```

Comment:
https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
Is it necessary to use docusaurus special syntax? Why not use `****` directly?

Reply:
Those contents are used in the docs site, this syntax will be rendered as `Admonitions Box`.

![image](https://user-images.githubusercontent.com/2106987/116953146-1a01f900-acbf-11eb-9e1e-2d1971f64114.png)